app/models/miq_region.rb

```diff
@@ -62,7 +62,7 @@ def miq_servers
   end
 
   def servers_for_settings_reload
-    miq_servers.where(:status => "started")
+    MiqServer.in_my_region.where(:status => "started")
```

Comment:
That's not accurate either.

For example, if we have current region 1 has server 1, and region 2 has server 2, and if someone makes a change to the settings for region 2, then this method will return server 1, which is wrong.  It should instead return `[]`.  I think you can do `miq_servers.in_my_region`, which, while a strange mix, is accurate (probably would warrant a code comment).  While it might be better to be explicit, and return a literal `[]`, I'd prefer to keep the method chainable.

Reply:
> `miq_servers.in_my_region` 

Hm this is what I had first but thought it was weird to be chaining `.in_region(region_number).in_my_region`

Reply:
:+1: done